### PR TITLE
type in the gap to applicable questions blockchain

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Marked if applied to all insights:

- [x] Remove all backticks from every headline (if found)

- [x] Add type in the gap and change context to standalone to all applicable questions

- [x] Remove mistakenly added type in the gap to True/False/Yes/No questions

- [x] Change some questions to support type in the gap

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/altcoins/altcoins.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/altcoins/altcoins.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/altcoins/why-is-it-worth-having-multiple-coins.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/altcoins/why-is-it-worth-having-multiple-coins.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/anonimity/anonymity.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/anonimity/anonymity.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/anonimity/how-to-de-anonymize-bitcoin.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/anonimity/how-to-de-anonymize-bitcoin.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/anonimity/mixing.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/anonimity/mixing.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/building-the-ledger/consensus.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/building-the-ledger/consensus.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/building-the-ledger/malicious-nodes.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/building-the-ledger/malicious-nodes.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/building-the-ledger/the-ledger.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/building-the-ledger/the-ledger.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/building-the-ledger/the-utxo-model.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/building-the-ledger/the-utxo-model.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/building-the-ledger/verifying-the-ledger-ii.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/building-the-ledger/verifying-the-ledger-ii.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/building-the-ledger/verifying-the-ledger-iii.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/building-the-ledger/verifying-the-ledger-iii.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/building-the-ledger/verifying-the-ledger.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/building-the-ledger/verifying-the-ledger.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/consensus-algorithms/byzantine-fault-tolerance.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/consensus-algorithms/byzantine-fault-tolerance.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/consensus-algorithms/consensus-algorithms.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/consensus-algorithms/consensus-algorithms.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/consensus-algorithms/delegated-proof-of-stake.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/consensus-algorithms/delegated-proof-of-stake.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/consensus-algorithms/proof-of-stake.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/consensus-algorithms/proof-of-stake.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/consensus-algorithms/proof-of-work.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/consensus-algorithms/proof-of-work.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/cryptoeconomics/cryptoeconomics.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/cryptoeconomics/cryptoeconomics.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/cryptoeconomics/how-do-cryptocurrencies-acquire-value.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/cryptoeconomics/how-do-cryptocurrencies-acquire-value.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/cryptoeconomics/why-is-the-price-fluctuating-so-much.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/cryptoeconomics/why-is-the-price-fluctuating-so-much.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/ethereum-i/ethereum-and-ether.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/ethereum-i/ethereum-and-ether.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/ethereum-i/what-are-smart-contracts.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/ethereum-i/what-are-smart-contracts.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/ethereum-i/what-does-ethereum-do-differently-ii.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/ethereum-i/what-does-ethereum-do-differently-ii.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/ethereum-i/what-does-ethereum-do-differently.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/ethereum-i/what-does-ethereum-do-differently.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/ethereum-ii/erc20-tokens.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/ethereum-ii/erc20-tokens.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/ethereum-ii/erc721-tokens.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/ethereum-ii/erc721-tokens.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/ethereum-ii/initial-coin-offerings.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/ethereum-ii/initial-coin-offerings.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/ethereum-ii/token-standards.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/ethereum-ii/token-standards.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/ethereum-on-the-web/metamask.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/ethereum-on-the-web/metamask.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/ethereum-on-the-web/myetherwallet.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/ethereum-on-the-web/myetherwallet.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/ethereum-on-the-web/web3js.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/ethereum-on-the-web/web3js.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/forking/forking-to-create-altcoins.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/forking/forking-to-create-altcoins.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/forking/forking-to-reverse-transactions.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/forking/forking-to-reverse-transactions.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/forking/what-are-forks.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/forking/what-are-forks.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/intro-blockchain/importance-of-blockchain.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/intro-blockchain/importance-of-blockchain.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/intro-blockchain/what-is-the-blockchain.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/intro-blockchain/what-is-the-blockchain.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/mining/energy-consumption-and-ecology.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/mining/energy-consumption-and-ecology.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/mining/how-can-one-mine.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/mining/how-can-one-mine.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/mining/mining-pools.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/mining/mining-pools.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/mining/mining.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/mining/mining.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/solidity/ether-heists-caused-by-bugs.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/solidity/ether-heists-caused-by-bugs.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/solidity/security-concerns.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/solidity/security-concerns.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/solidity/what-is-solidity.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/solidity/what-is-solidity.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/the-bitcoin-blockchain/bitcoin.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/the-bitcoin-blockchain/bitcoin.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/the-bitcoin-blockchain/the-bitcoin-block.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/the-bitcoin-blockchain/the-bitcoin-block.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/the-bitcoin-blockchain/the-bitcoin-blockchain.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/the-bitcoin-blockchain/the-bitcoin-blockchain.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/the-blockchain/blockchain-components.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/the-blockchain/blockchain-components.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/the-blockchain/blocks-ii.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/the-blockchain/blocks-ii.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/the-blockchain/blocks-iii.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/the-blockchain/blocks-iii.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/the-blockchain/blocks.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/the-blockchain/blocks.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/wallets/identity.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/wallets/identity.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/wallets/keeping-your-wallet-secure.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/wallets/keeping-your-wallet-secure.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)
- [ ] [enkidevs/curriculum/blockchain/blockchain-fundamentals/wallets/wallets.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/blockchain/blockchain-fundamentals/wallets/wallets.md?ref=573d1b5c60c8fae5105ddc2e5519220042cdf5b4)

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| the-bitcoin-block | <ul><li>`First block mined` pointing to `https://blockchain.info/block/000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f` is broken because `HTTP_404`</li></ul> |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| the-bitcoin-block | <ul><li>`First block mined` pointing to `https://www.blockchain.com/btc/block/000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f` is broken because `HTTP_404`</li></ul> |

> If you notice anything wrong with these rules contact andrei@enki.com.

File: blockchain/blockchain-fundamentals/ethereum-i/what-does-ethereum-do-differently-ii.md [hunk 1]
```diff
@@ -5,11 +5,12 @@ category: must-know
 practiceQuestion:
   formats:
     - fill-in-the-gap
-  context: relative
+    - type-in-the-gap
```
Comment (kapnobatai137) [inline blockchain/blockchain-fundamentals/ethereum-i/what-does-ethereum-do-differently-ii.md:8]:
```suggestion
```

File: blockchain/blockchain-fundamentals/the-bitcoin-blockchain/the-bitcoin-blockchain.md [hunk 2]
```diff
@@ -5,11 +5,12 @@ category: must-know
 practiceQuestion:
   formats:
     - fill-in-the-gap
-  context: relative
+    - type-in-the-gap
```
Comment (kapnobatai137) [inline blockchain/blockchain-fundamentals/the-bitcoin-blockchain/the-bitcoin-blockchain.md:8]:
```suggestion
```

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]

**Review by kapnobatai137 — APPROVED** [review]

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| the-bitcoin-block | <ul><li>`First block mined` pointing to `https://www.blockchain.com/btc/block/000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f` is broken because `HTTP_404`</li></ul> |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Pull request merged**